src/pages/ViewScholarship.jsx

```diff
@@ -51,38 +52,41 @@ export default function ViewScholarship() {
 
   const { name, dateAdded, lastModified } = scholarship.data;
   return (
-    <Container maxWidth="md">
-      <Helmet>
-        <title>{name}</title>
-      </Helmet>
+    <>
+      <Container maxWidth="md">
+        <Helmet>
+          <title>{name}</title>
+        </Helmet>
 
-      {justEdited && navType === 'PUSH' && (
-        <Collapse in={showAlert}>
-          <Alert
-            color="primary"
-            variant="filled"
-            onClose={() => setShowAlert(false)}>
-            {`Scholarship successfully ${
-              Date.parse(dateAdded) === Date.parse(lastModified)
-                ? 'submitted.'
-                : 'updated.'
-            }`}
-          </Alert>
-        </Collapse>
-      )}
+        {justEdited && navType === 'PUSH' && (
```

Comment:
not sure why these spaces are present but you might want to use [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) to avoid this in the future